res/controllers/Hercules DJ Console 4-Mx.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
```

Comment:
Shouldn't Mixxx refuse to load a mapping if the minimum compatible version number of the mapping is higher than the running version of Mixxx?